pkg/commands/workload.go

```diff
@@ -494,6 +513,56 @@ func (opts *WorkloadOptions) PublishLocalSource(ctx context.Context, c *cli.Conf
 	return okToPush, nil
 }
 
+func (opts *WorkloadOptions) PublishLocalSourceWithoutPrompts(ctx context.Context, c *cli.Config, currentWorkload, workload *cartov1alpha1.Workload) error {
```

Comment:
I understand the intention here is to skip printing anything so that `--output` does not include any text. If that is the case, instead of having a separate function which is mostly a replica of PushLocalSource have a utility function with an indicator to either print or silent return. That way we don't have to maintain two implementations.

Reply:
Thanks Ani, 

I changed it to be one function that would check in every step if there should be any prompts